src/modules/fancyzones/FancyZonesLib/ZoneSet.cpp

```diff
@@ -348,6 +348,40 @@ ZoneSet::MoveWindowIntoZoneByIndexSet(HWND window, HWND workAreaWindow, const Zo
 
     if (!sizeEmpty)
     {
+        auto zoneTitleBar = m_zoneTitleBarByIndexSets.find(indexSet);
+
+        // Are we are not alone?
+        if (!m_windowsByIndexSets[indexSet].empty())
+        {
+            if (zoneTitleBar == m_zoneTitleBarByIndexSets.end())
+            {
+                // Create a new zone title bar
```

Comment:
Is this comment obsolete?

Reply:
The comment refers to the entire block of code (starting in line 358)